packages/backend-core/src/queue/inMemoryQueue.ts

```diff
@@ -164,13 +174,14 @@ class InMemoryQueue implements Partial<Queue> {
    */
   async close() {}
 
-  /**
-   * This removes a cron which has been implemented, this is part of Bull API.
-   * @param cronJobId The cron which is to be removed.
-   */
-  async removeRepeatableByKey(cronJobId: string) {
-    // TODO: implement for testing
-    console.log(cronJobId)
+  async removeRepeatableByKey(id: string) {
+    for (const [idx, message] of this._messages.entries()) {
+      if (message.opts?.jobId?.toString() === id) {
+        this._messages.splice(idx, 1)
+        this._emitter.emit("removed", message)
+        return
+      }
+    }
```

Comment:
This was put in to try and test us removing repeatable jobs from Bull but in practice it never gets called, due to my change in how crons get triggered. Because we trigger them manually, the job message doesn't have a `job.opts.repeat` anymore. I couldn't figure a way around this, we can't easily mimic Bull's repeat job functionality in our `InMemoryQueue` class.